src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import AddTaskButton from "../../Atoms/AddTaskButton";
+import Task from "../../Molecules/Task";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+
+const TodoCard = () => {
+  const [task, setNewTask] = useState([]);
+  const pushAddbutton = () => {
+    setNewTask([...task, { name: "", state: "TODO" }]);
+  };
+  const editingTask = task.map(({ name, state }, index) => {
+    if (state == "TODO") {
+      return (
+        <Task
+          onEditComplete={(taskName) => {
+            let editedTask = [...task];
```

Comment:
```suggestion
            const editedTask = [...task];
```
constで宣言された変数への再宣言、再代入は不可能ですが、プロパティの編集は可能です。
そのため、今回の処理であればconstで大丈夫です。
基本的に let より const の方が望ましいということを意識できるとグッドだと思います！
（些細な点なので、今回は直さなくても大丈夫です。）